File: cadastro.py

class cadastro:
    def __init__(self, nome, númerodetelefone, datadenascimento, CEP): #iniciando uma classe
        self.nome = nome
        self.númerodetelefone = númerodetelefone
        self.datadenascimento = datadenascimento
        self.CEP = CEP
        
cadastros = []
#criamos um loop para adicionar cadastros

while 'true':
    resposta = input ("deseja adicionar cadastro? (s/n): ")
    if resposta.lower()!= '':


        #para ler as informações do cadastro
        nome = input("Digite o nome: ")
        númerodetelefone = str(input("Digite seu número de telefone: "))
        datadenascimento = str(input("Digite a data de nascimento: "))
        CEP = str(input("Digite seu CEP: "))

        #criamos um dicionario para armazenar as informações do cadastro 
        cadastro_str = {"nome", "númerodetelefone", "datadenascimento", "CEP"}

        #para adicionar o cadastro ao vetor
        cadastros.append(cadastro)
        
        arquivo = open("clientes.txt", "a")

        arquivo.write(cadastro_str)
        
        print ("Cadastro adicionado com sucesso!")
        
        break

File: cadastro.js
import java.util.Scanner;

public class Cadastro {
    private String nome;
    private String numero;
    private String data de nascimento;
    private String cep;

    public Cadastro() {}

    public void cadastrar() {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite seu nome: ");
        nome = scanner.nextLine();

        System.out.print("Digite seu numero: ");
        email = scanner.nextLine();

        System.out.print("Digite sua data de nascimento: ");
        senha = scanner.nextLine();

        System.out.print("Digite seu cep: ");
        cpf = scanner.nextLine();

        System.out.println("Cadastro realizado com sucesso!");
        System.out.println("Dados cadastrados:");
        System.out.println("Nome: " + nome);
        System.out.println("numero: " + numero);
        System.out.println("data de nascimento: " + data de nascimento);
        System.out.println("cep: " + ce);
    }

    public static void main(String[] args) {
        Cadastro cadastro = new Cadastro();
        cadastro.cadastrar();
    }
}
